evproc/processor.py

```diff
@@ -13,6 +13,7 @@
 #    express or implied. See the License for the specific language
 #    governing permissions and limitations under the License.
 
+import collections
 import stevedore
```

Comment:
Put a blank line after the "import collections"; I try to conform to OpenStack style, which has 3 blocks of imports, separated by blank lines: standard library imports (e.g., collections); third-party imports (e.g., stevedore); and internal imports (e.g., evproc.exc).